fs/sync/sync.go

```diff
@@ -427,8 +431,12 @@ func deleteEmptyDirectories(f fs.Fs, entries fs.DirEntries) error {
 		return nil
 	}
 	if accounting.Stats.Errored() {
-		fs.Errorf(f, "%v", fs.ErrorNotDeletingDirs)
-		return fs.ErrorNotDeletingDirs
+		if fs.Config.IgnoreErrors {
```

Comment:
same here...

Note that there are some more uses of `fs.ErrorNotDeleting` and `fs.ErrorNotDeletingDirs` which need to be addressed too.

Reply:
Yes those lines need changing too.